lib/PlaidClient.js

```diff
@@ -210,6 +211,44 @@ Client.prototype.getTransactions =
     }, options, cb);
   };
 
+// getAllTransactions(String, Date, Date, Object?, Function)
+Client.prototype.getAllTransactions =
+  function(access_token, start_date, end_date, cb) {
+    var transactions = [];
+    var offset = 0, total_transactions = null;
+    var self = this;
+
+    var handleResponse = function(transactionsResponse) {
+      total_transactions = transactionsResponse.total_transactions;
```

Comment:
the documentation is somewhat ambiguous, but I believe that this represents the total transactions available _within the date range_ (e.g. that this is the proper way to terminate the pagination).

cc @michaelckelly @pbernasconi we should consider clarifying the documentation here: https://plaid.com/docs/api/#transactions